lib/filter-tile/index.js

```diff
@@ -17,12 +17,14 @@ export default class FilterTile extends Component {
     this.setState({infoBoxVisible: !this.state.infoBoxVisible});
   }
   render () {
-    const { description, color, onYesFilter, onNoFilter, filterVisible, showAddMessage } = this.props;
+    const { description, onYesFilter, onNoFilter, showAddMessage } = this.props;
     const { infoBoxVisible } = this.state;
+    const type = description.tagid.split(':')[0];
+    const color = (type === 'geo' ? '#CEC947' : '#8EB8C4');
```

Comment:
This is copied from the tag component. We may want to separate it out into a common module.